internal/learn/about-iota/data-transfer.md

```diff
@@ -149,19 +158,17 @@ A message may contain a payload. Three payload types are currently defined in th
 
 | Payload Name                                                                                                    | Type Value |
 | --------------------------------------------------------------------------------------------------------------- | ---------- |
-| [Transaction Payload](https://github.com/iotaledger/tips/blob/main/tips/TIP-0007/tip-0007.md)                   | 0          |
-| [Milestone Payload](https://github.com/iotaledger/tips/blob/main/tips/TIP-0008/tip-0008.md)                     | 1          |
-| [Indexation Payload](https://github.com/iotaledger/tips/blob/main/tips/TIP-0006/tip-0006.md#indexation-payload) | 2          |
+| [Tagged Date](https://hackmd.io/TIP-0023/tip-0023.md)                                                           | 5          |
+| [Transaction](https://hackmd.io/TIP-0020/tip-0020.md)                                                           | 6          |
+| [Milestone](https://hackmd.io/TIP-0029/tip-0029.md)                                                             | 7          |
 
 A message containing only an **indexation payload** (Data) can be sent without a signature. It can hold any data the user wants to send, as long as it is parsable and follows the required syntax and size limit.
```

Comment:
```suggestion
A message containing only a **tagged data payload** (Data) can be sent without a signature. It can hold any data the user wants to send, as long as it is parsable and follows the required syntax and size limit.
```